File: bot/src/main/java/com/lm_pakkanen/radio_pele_java/models/message_embeds/QueueShuffledEmbed.java
package com.lm_pakkanen.radio_pele_java.models.message_embeds;

import com.lm_pakkanen.radio_pele_java.interfaces.IEmbedBuilder;
import lombok.Getter;
import net.dv8tion.jda.api.EmbedBuilder;
import net.dv8tion.jda.api.entities.MessageEmbed;

@Getter
public final class QueueShuffledEmbed implements IEmbedBuilder {

  private final MessageEmbed embed;

  public QueueShuffledEmbed() {
    final EmbedBuilder embedBuilder = IEmbedBuilder.getEmbedBuilder();
    embedBuilder.setTitle("Q shuffled.");
    this.embed = embedBuilder.build();
  }

}

File: bot/src/main/kotlin/com/lm_pakkanen/radio_pele_java/models/message_embeds/QueueShuffledEmbed.kt
package com.lm_pakkanen.radio_pele_java.models.message_embeds

import com.lm_pakkanen.radio_pele_java.interfaces.IEmbedBuilder
import net.dv8tion.jda.api.entities.MessageEmbed

class QueueShuffledEmbed : IEmbedBuilder {
    override val embed: MessageEmbed

    init {
        val embedBuilder = IEmbedBuilder.getEmbedBuilder()
        embedBuilder.setTitle("Q shuffled.")
        this.embed = embedBuilder.build()
  }
}
